starlite/openapi/controller.py

```diff
@@ -58,13 +59,40 @@ def get_schema_from_request(request: Request) -> "OpenAPI":
             An [OpenAPI][pydantic_openapi_schema.v3_1_0.open_api.OpenAPI] instance.
 
         Raises:
-            [ImproperlyConfiguredException][starlite.exceptions.ImproperlyConfiguredException]: If the application
-                `openapi_schema` attribute is `None`.
+            ImproperlyConfiguredException: If the application `openapi_config` attribute is `None`.
         """
         if not request.app.openapi_schema:  # pragma: no cover
             raise ImproperlyConfiguredException("Starlite has not been instantiated with OpenAPIConfig")
         return request.app.openapi_schema
 
+    def should_serve_endpoint(self, request: Request) -> bool:
+        """This method verifies that the requested path is within the enabled
+        endpoints in the openapi_config.
+
+        Args:
+            request: To be tested if endpoint enabled.
```

Comment:
So, since we are generating the reference documentation from the doc strings, we need to follow the `mkdocstrings` notation here. This means that this needs to be updated to include the linkage, e.g.

```suggestion
            request: The [Request][starlite.connection.Request] instance.
```

Reply:
These too, I suggested we don't need to cross-ref the type of the arg in the description as the docs already work it out:

![image](https://user-images.githubusercontent.com/20659309/187645576-b460029c-0eef-492b-8b06-37b6c188ce17.png)